simpeg_drivers/components/meshes.py

```diff
@@ -114,6 +123,30 @@ def _initialize(self) -> None:
         self.uid = self.entity.uid
         self.n_cells = self.entity.n_cells
 
+    def _auto_mesh(self):
+        """Automate meshing based on data and topography objects."""
+
+        params = auto_mesh_parameters(
+            self.params.data_object, self.params.topography_object
+        )
+        self._mesh = OctreeDriver.treemesh_from_params(params)
+
+        mesh_group = UIJsonGroup.create(
+            self.workspace, name="AutoMesh", parent=self.params.out_group
+        )
+        ifile = params.input_file
+        ifile.write_ui_json(name="AutoMesh")
```

Comment:
No update done on the ifile data. And we currently have a bug that prevents us from write it out from data - the refinements never get converted from list[int] to str

Reply:
Cleaned out the options saving.  To be addressed after GEOPY-1543